framework/src/org/checkerframework/framework/stub/StubParser.java

```diff
@@ -858,7 +858,12 @@ private void parseField(
         // StubParser parses all annotations in type annotation position as type annotations
         annotateDecl(declAnnos, elt, decl.getElementType().getAnnotations());
         AnnotatedTypeMirror fieldType = atypeFactory.fromElement(elt);
-        annotate(fieldType, decl.getElementType());
+        // Getting type from the first variable in a line
+        // (all the variables have the same type as they written in one line of code).
```

Comment:
Can you clarify what the difference between `decl.getElementType()` and the new code is?
Why is the type of the field declaration not the same as the type of the first variable in the field declaration?
The comment about `same type ... in one line of code` is also not correct: `@Foo int @Bar [] a, b @Baz [] , c @Nope [];` is a valid field declaration. `@Foo` should be applied to `int` in all three arrays.